misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -149,3 +151,10 @@ data class CorsConfig(
   /** A comma separated list of HTTP headers that are allowed to be exposed on the client. */
   val exposedHeaders: Array<String> = arrayOf()
 )
+
+data class AdminDashboardConfig(
```

Comment:
I think this should be a completely separate set of config in `misk.web.metadata`, since `WebConfig` deals with the more base-level web server config.

Alternatively, thinking about this a bit more, this could really just be configured at the module, skipping the yaml reading altogether.

Reply:
Do you have an example of what you mean by "configured at the module"?

Reply:
https://github.com/cashapp/misk/blob/master/misk/src/test/kotlin/misk/web/metadata/MetadataTestingModule.kt, used in the dashboard tests, binds a DashboardHomeUrl among other data classes at that point. Following that pattern, something like:
```kotlin
bind<ProtobufDefinitionUrlPrefix>().toInstance(ProtobufDefinitionUrlPrefix("http://example.com")
```